### PR TITLE
Add README.md file

### PR DESCRIPTION
**Pull request by irisyad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# github-profile-readme
\ No newline at end of file
+# github-profile-readme
+![logo.jpeg](src/assets/logo.jpeg)
+### Let's create a GitHub profile README :zap:
```
Comment (irisyad) [inline README.md:3]: [ Sand ] Better to use README

File: src/streamlit_app.py [hunk 2]
```diff
@@ -25,9 +26,11 @@
     }
 )
 
+image1 = Image.open('../src/assets/profile-with-readme.png')
```
Comment (irisyad) [inline src/streamlit_app.py:29]: [Boulder] Use open() from image module in PIL to open the image

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+streamlit==1.17.0
```
Comment (irisyad) [inline requirements.txt:1]: [sand] Better to add Streamlit version

**Comment by irisyad** [comment]
- Add update for README 
- Add Streamlit version
- Open image file with PIL

File: src/github_profile.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 def generate_profile(theme, **kwargs):
     """Generate profile readme from theme and kwargs"""
     # Read theme profile
-    with open(f"src/themes/{theme}/profile.txt") as f:
+    with open(f"../src/themes/{theme}/profile.txt") as f:
```
Comment (r-dehghani) [inline src/github_profile.py:7]: ⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?

File: src/github_profile.py [hunk 5]
```diff
@@ -15,7 +15,7 @@ def generate_profile(theme, **kwargs):
             continue
 
         # Skip items that don't exist in theme
-        item_path = Path(f"src/themes/{theme}/{item}.txt")
+        item_path = Path(f"../src/themes/{theme}/{item}.txt")
```
Comment (r-dehghani) [inline src/github_profile.py:18]: ⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?

File: src/streamlit_app.py (hunk 2 above)
Comment (r-dehghani) [inline src/streamlit_app.py:29]: ⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?

File: src/streamlit_app.py [hunk 6]
```diff
@@ -69,7 +72,7 @@
 '''
 
 # Select Theme
-themes = Path('src/themes').iterdir()
+themes = Path('../src/themes').iterdir()
```
Comment (r-dehghani) [inline src/streamlit_app.py:75]: ⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?

**Review by r-dehghani — COMMENTED** [review]
⚪ I think if the `../` cab be deleted from the path and run the application on top of the src folder.

**Pull request closed without merging**